jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -61,14 +60,21 @@ def on_limits_change(self, *args):
 
 
 @viewer_tool
-class BlinkOnce(Tool):
+class BlinkOnce(CheckableTool):
     icon = os.path.join(ICON_DIR, 'blink.svg')
     tool_id = 'jdaviz:blinkonce'
     action_text = 'Go to next image'
-    tool_tip = ('Click on this button to display the next image, '
+    tool_tip = ('Click on the viewer to display the next image, '
                 'or you can also press the "b" key anytime')
 
     def activate(self):
+        self.viewer.add_event_callback(self.on_click,
+                                       events=['click'])
+
+    def deactivate(self):
```

Comment:
I changed the blink tool to be a "checkable" tool that blinks-on-viewer-click instead of blink-on-tool-click so it would be consistent with the other tools in the same group, and so that I could have different behavior for the plugin shortcuts.... but we could of course revert that if anyone has strong opinions against it.